code/game/gamemodes/miniantags/bot_swarm/swarmer.dm

```diff
@@ -155,30 +155,12 @@
 	S.DisIntegrate(src)
 	toggle_cam(S, 0)
 
-/obj/machinery/particle_accelerator/swarmer_act(mob/living/simple_animal/hostile/swarmer/S)
+/obj/structure/particle_accelerator/swarmer_act(mob/living/simple_animal/hostile/swarmer/S)
```

Comment:
Test this out, it you would, please---as on as it is, @Aurorablade had to add each one because just having this single one didn't exempt all parts, for whatever reason.